docs-next/pages/guides/cli.mdx

```diff
@@ -26,25 +25,17 @@ The CLI supports commands in three categories; Run, Build and Migrate.
 ### Run
 
 The **run** commands are used to prepare and then start your Keystone server.
-Keystone can be run in three different modes; **prototyping**, **dev** and **production**.
-These different modes support different phases of your project life-cycle.
-The different modes differ in how they interact with your database and with your Admin UI application.
+Keystone can be run in two different modes; **dev** and **production**.
 
 Note: All the **run** commands expect to find a module called `keystone.ts` with a default export which returns a system configuration `config()` from `@keystone-next/keystone/schema`.
 See the [System Configuration API](../apis/config) for details on how to configure a Keystone system.
 
-#### keystone-next prototype (default)
+#### keystone-next dev (default)
 
-In **prototyping** mode, Keystone will try its hardest to put your database into a state which is consistent with your schema.
-This might require Keystone to delete data in your database.
-This mode of operation should only be used when you are first getting started with Keystone and are not yet working with real data.
-In prototyping mode you can quickly change your schema and immediately see the changes reflected in your database and Admin UI when you restart.
+In **dev** mode, Keystone will start the dev server and update your database is a different way depending on `db.useMigrations`:
 
-#### keystone-next dev
-
-**Dev** mode is identical to **prototyping** mode when using the `knex` of `mongoose` adapters.
-When using the `prisma` adapter, Keystone will use Prisma's migration framework to generate and locally apply migrations when you start your system.
-If you are using `prisma` and your database already includes migrations then you must use **dev** mode, and not **prototyping** mode.
+- If `db.useMigrations` is false(the default), Keystone will use Prisma Migrate to update your databse so that it matches your schema. It may lose data while updating your databse so you should only use this mode in initial development.
```

Comment:
```suggestion
- If `db.useMigrations` is `false` (the default), Keystone will use Prisma Migrate to update your database so that it matches your schema. It may lose data while updating your database so you should only use this mode in initial development.
```